crates/bin/src/args.rs

```diff
@@ -278,6 +291,18 @@ impl Default for RateLimit {
     }
 }
 
+/// Strategy for installing the package
+#[derive(Debug, Copy, Clone, Eq, PartialEq, Ord, PartialOrd, ValueEnum)]
+pub enum Strategy {
+    /// Attempt to download official pre-built artifacts using
+    /// information provided in `Cargo.toml`.
+    Release,
```

Comment:
Not a fan of using `release` here, as it's way too vague / could mean lots of things.

Reply:
Could go with `official`? given the description?

I wonder if we could meaningfully split `official` and `guess`, actually. With `official` being only if there's explicit binstall metadata in the Cargo.toml, and `guess` otherwise (which would still look in the toml for the repository and such).

Reply:
I renamed that to `CrateMetaData` since `MetaData` sounds a little bit vague.